tirocks-sys/bindings/bindings.rs

```diff
@@ -0,0 +1,5481 @@
+        column_families: *mut *mut crocksdb_column_family_handle_t,
+        iterators: *mut *mut crocksdb_iterator_t,
+        size: usize,
+        errptr: *mut *mut libc::c_char,
+    );
+}
+extern "C" {
+    pub fn ctitandb_delete_files_in_range(
+        db: *mut crocksdb_t,
+        start_key: *const libc::c_char,
+        start_key_len: usize,
+        limit_key: *const libc::c_char,
+        limit_key_len: usize,
+        include_end: libc::c_uchar,
+        errptr: *mut *mut libc::c_char,
+    );
+}
+extern "C" {
+    pub fn ctitandb_delete_files_in_range_cf(
+        db: *mut crocksdb_t,
+        column_family: *mut crocksdb_column_family_handle_t,
+        start_key: *const libc::c_char,
+        start_key_len: usize,
+        limit_key: *const libc::c_char,
+        limit_key_len: usize,
+        include_end: libc::c_uchar,
+        errptr: *mut *mut libc::c_char,
+    );
+}
+extern "C" {
+    pub fn ctitandb_delete_files_in_ranges_cf(
+        db: *mut crocksdb_t,
+        cf: *mut crocksdb_column_family_handle_t,
+        start_keys: *const *const libc::c_char,
+        start_keys_lens: *const usize,
+        limit_keys: *const *const libc::c_char,
+        limit_keys_lens: *const usize,
+        num_ranges: usize,
+        include_end: libc::c_uchar,
+        errptr: *mut *mut libc::c_char,
+    );
+}
+extern "C" {
+    pub fn ctitandb_delete_blob_files_in_range(
+        db: *mut crocksdb_t,
+        start_key: *const libc::c_char,
+        start_key_len: usize,
+        limit_key: *const libc::c_char,
+        limit_key_len: usize,
+        include_end: libc::c_uchar,
+        errptr: *mut *mut libc::c_char,
+    );
+}
+extern "C" {
+    pub fn ctitandb_delete_blob_files_in_range_cf(
+        db: *mut crocksdb_t,
+        column_family: *mut crocksdb_column_family_handle_t,
+        start_key: *const libc::c_char,
+        start_key_len: usize,
+        limit_key: *const libc::c_char,
+        limit_key_len: usize,
+        include_end: libc::c_uchar,
+        errptr: *mut *mut libc::c_char,
+    );
+}
+extern "C" {
+    pub fn ctitandb_delete_blob_files_in_ranges_cf(
+        db: *mut crocksdb_t,
+        cf: *mut crocksdb_column_family_handle_t,
+        start_keys: *const *const libc::c_char,
+        start_keys_lens: *const usize,
+        limit_keys: *const *const libc::c_char,
+        limit_keys_lens: *const usize,
+        num_ranges: usize,
+        include_end: libc::c_uchar,
+        errptr: *mut *mut libc::c_char,
+    );
+}
+#[repr(u32)]
+#[doc = " Keep adding ticker's here."]
+#[doc = "  1. Any ticker should be added before TICKER_ENUM_MAX."]
+#[doc = "  2. Add a readable string in TickersNameMap below for the newly added ticker."]
+#[doc = "  3. Add a corresponding enum value to TickerType.java in the java API"]
+#[doc = "  4. Add the enum conversions from Java and C++ to portal.h's toJavaTickerType"]
+#[doc = " and toCppTickers"]
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
+pub enum rocksdb_Tickers {
```

Comment:
can we have a way to add a prefix `c` for these enums to make them consistent with other types

Reply:
or cancel the prefix `c` in the `crocksdb.h`

Reply:
No, we can't. https://github.com/rust-lang/rust-bindgen/issues/1098. I think the c prefix is fine as it can tell what types are defined by ourselves and what types are defined by rocksdb itself.